### PR TITLE
#89 Added script to generate pkg hex icons

### PR DESCRIPTION
**Pull request by jwang-lilly**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
@jwang-lilly wow, this is super cool! I tested it out for admiral and it works very well. I think we are almost ready to merge - can you please just add an entry in the README as well, perhaps under "What is included in this template" to alert users that this is present?

**Comment by manciniedoardo** [comment]
@bms63 @bundfussr FYI

**Comment by bms63** [comment]
> @jwang-lilly wow, this is super cool! I tested it out for admiral and it works very well. I think we are almost ready to merge - can you please just add an entry in the README as well, perhaps under "What is included in this template" to alert users that this is present?

@manciniedoardo do we need to clean up the favicons in admial?

**Review by bms63 — COMMENTED** [review]
Might be nice touch to put into changelog!!

File: NEWS.md [hunk 1]
```diff
@@ -6,3 +6,8 @@ Note that after first release you should then start maintaining this `NEWS.md` f
 You should categorize the updates under title sections for New Features, Updates of Existing Functions,
 Breaking Changes, Documentation, Various.
 Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+
+
+## New Features
+
+- Tool to generate Apple touch icon and favicon given a package logo using generate_pkg_favicons() (#89)
```
Comment (manciniedoardo) [inline NEWS.md]:
```suggestion
- Added a utility function `generate_pkg_favicons()` under `inst/favicon_tools` to help  generate Apple touch icons and favicons given a package logo. (#89)
```

File: NEWS.md [hunk 2]
```diff
@@ -6,3 +6,8 @@ Note that after first release you should then start maintaining this `NEWS.md` f
 You should categorize the updates under title sections for New Features, Updates of Existing Functions,
 Breaking Changes, Documentation, Various.
 Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+
+
+## New Features
+
+- Added a utility function `generate_pkg_favicons()` under `inst/favicon_tools` to help  generate Apple touch icons and favicons given a package logo. (#89)
```
Comment (manciniedoardo) [inline NEWS.md]:
```suggestion
- Added a utility function `generate_pkg_favicons()` under `inst/favicon_tools` to help generate Apple touch icons and favicons given a package logo. (#89)
```

**Comment by manciniedoardo** [comment]
Finally getting round to merging this - thanks @jwang-lilly !

**Review by manciniedoardo — APPROVED** [review]
